README.md

```diff
@@ -0,0 +1,157 @@
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Ruby Capstone Project ](#-ruby-capstone-project-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites ](#prerequisites-)
+    - [Setup ](#setup-)
+    - [Install ](#install-)
+    - [Usage ](#usage-)
+  - [👥 Authors ](#-authors-)
+    - [👤 **Josue Ruhanika**](#-Josue-Rushanika)
+    [👤 **Obed Bamfo**](#-Obed Bamfo)
+  
+  - [Presentation Video ](#presentation-video-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Ruby Capstone Project <a name="about-project"></a>
+
+**Ruby Capstone Project** is an console app designed with Ruby where you can add a music album, movie, books, and games borrowing records.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>Ruby</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **User can add a music album**
+- **User can add a movie**
+- **User can add a book**
+- **User can add a game**
+- **Library Record Keeping**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites <a name="prerequisites"></a>
+
+In order to run this project you need the following installed in your local system
+
+- Ruby
+- Git CLI
+- Github account
+
+### Setup <a name="setup"></a>
+
+Clone this repository to your desired folder:
+
+```sh
+  cd <desired-folder>
+  git clone https://github.com/josuerushanika/ruby_capstone.git
+```
+
+### Install <a name="install"></a>
+
+```sh
+  cd Ruby-Capstone-Project
+  bundle install
+```
+
+### Usage <a name="usage"></a>
+
+```sh
+  ruby main.rb
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+### 👤 **Josue Rushanika**
+
+- GitHub: [@githubhandle](https://github.com/josuerushanika)
+- Twitter: [@twitterhandle](https://twitter.com/josuerushanika)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/josue-rushanika-336049b3/)
+
+
+### 👤 ** Obed Bamfo **
+
+- GitHub: [@obed300](https://github.com/obed300)
+- Twitter: [@obed300](https://twitter.com/bamfo_obed)
+- LinkedIn: [@obed300](https://www.linkedin.com/in/obed-bamfo-4b152421b/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- PRESENTATION VIDEO -->
+
+## Presentation Video <a name="presentation-video"></a>
+
+[Video](https://drive.google.com/file/d/1DTcLKxU8SM81VKbKYnSmqoNTlY-_RWGt/view)
+
```

Comment:
- [ ] Great job recording a video, I was happy to see both of you appear and highlight the project pieces. However, there are a few requirements we need to meet for the video to reach an acceptable standard. Kindly record another video with a maximum of __5 minutes__. In order to meet this requirement, you should avoid one person sharing the screen. Each one of you should share your work (the first driver stops screen sharing and the second person starts sharing).

![video](https://github.com/josuerushanika/ruby_capstone/assets/53124548/17471fa2-9f3b-405d-ba86-26dc3402f455)

Reply:
@elisha2kyakpo1 , thank you. We have completed another video presentation.